cypress/integration/footerSpec.js

```diff
@@ -0,0 +1,78 @@
+import {
+  checkLinkStyling,
+  checkFooterLinks,
+  getElement,
+  shouldContainText,
+  shouldContainStyles,
+} from '../support/testHelper';
+
+describe('Footer Tests', () => {
+  // eslint-disable-next-line no-undef
+  before(() => {
+    // Only 'c0000000025o' & 'c0000000027o' are available within the PROD enviroment
+    cy.visit('/news/articles/c0000000025o');
+  });
+  it('should render the BBC News branding', () => {
```

Comment:
Aren't these tests essentially just what is tested on the header? We should re-use all of the code.

Reply:
Yes, they're the same, other than the element name. Agreed on re-using the existing code.